src/main/java/stepsdesign/CdfHubSteps.java

```diff
@@ -41,6 +41,10 @@ public static void openHub() {
   public static void openPlugin(String pluginName) {
     CdfHubActions.clickOnPlugin(pluginName);
   }
+  @Then("Click on {string} pipeline")
+  public static void openPipeline(String pipelineName) {
+    CdfHubActions.clickOnPlugin(pipelineName);
```

Comment:
Is this a plugin or pipeline? Please use proper naming.

Reply:
done